### ISSUE TITLE
Recipe Drawer (1x1) from Functional Storage not available

### ISSUE THREAD

**Issue by Bolukan** [FTBTeam/FTB-Modpack-Issues#4145]
### Modpack

FTB Presents Direwolf20 1.20

### Modpack version

1.4.0

### Log Files

n.a.

### Describe the bug

The recipe for the Oak en Birch Drawer (1x1) from the mod Functional Storage can't be used to create the item as the same recipe is used to create three Oak, respectively Birch Cupboard of the mod Handcrafted.
Vice versa the recipe for three Jungle Cupboard is not available as it is used for the Jungle Drawer (1x1).

I expect the 11 variants of cupboards and 10 variants of Drawer (1x1) are the same.

The recipe is a chest in the middle surrounded with 8 planks

### Steps to reproduce

Try via JEI, or just put the chest in the middle and 8 planks around and try to produce both items with the same recipe.

### Expected behaviour

Both items have their own unique recipe

### Screenshots

n.a.

### Additional information

n.a.

**Comment by decryptedchaos** [comment]
came here to report the same issue it's been driving me crazy and there's no conflict solver mods for 1.20

**Comment by Medivha** [comment]
You can choose between the 2 but only with the normal Crafting Bench. All other crafting grids don't have it implemented(and randomly pick one recipe).
![javaw_2023-11-01_16-13-46](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/16429350/68905a5f-1cf2-4954-9a69-e6097c75faaa)

yeah idk if the mod that adds that button should implement this for most other mods or the other mods do the same or change the recipes. Not the first time we get conflicting recipes and people don't know the normal bench thing because nobody use it at a certain point.

**Comment by Bolukan** [comment]
Did not know this. Thanks

**Comment by decryptedchaos** [comment]
>You can choose between the 2 but only with the normal Crafting Bench. All other crafting grids don't have it implemented(and randomly pick one recipe).

Not a bad workaround.  

However the one thing I'll say is the recipe conflicts are becoming alarmingly unmanageable (i'm looking at recipes for individual colored items too"   

This all is most notable when doing heavy auto crafting..  a favorite pastime of mine,  the system can't deal with conflicts well ore dict has tried to solve some of that and for some of it it works decent enough refined storage does a pretty good job of substituting things that are interchangeable 

But when 2 - 3 mods all uses the same exact recipe there is no way for it to choose which one,  so adding the button might work for manual crafting I'm not sure if it can work for auto crafting..  because you'd have to somehow store the state of the recipe in the crafting pattern. 

Something that would tell it use 
- crafting id 0 here
- crafting id 1 there

So for something like that to work,  i would expect any mod that crafts items programmatically would somehow have to hook that function.

**Comment by Medivha** [comment]
I agree, there should be no conflicting recipes in the first place. It creates too many problems/ confusion.
Autocrafting should be fine because you also choose the outcome, not only the materials. Of course, you have to pay attention to that ;), and we are back to it's better to not have same recipes.

**Comment by dalgrim** [comment]
Why not change the recipe to appropriate wood slabs on left and right center row? That would make the recipe unique.

**Comment by localzuk** [comment]
This is an issue if you use Refined Storage, as the button to choose between recipe outputs doesn't appear if you are creating a pattern for auto-crafting purposes.

![image](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/998988/1b820eb9-cbfc-468b-ae7b-655e0dff8aa3)

**Comment by Medivha** [comment]
yeah looks like the pattern is not that smart and only cares about the recipe not the outcome.
